Gems/LyShine/Code/Editor/Animation/UiAnimViewAnimNode.cpp

```diff
@@ -1482,7 +1482,7 @@ bool CUiAnimViewAnimNode::PasteNodesFromClipboard(QWidget* context)
     const bool bLightAnimationSetActive = GetSequence()->GetFlags() & IUiAnimSequence::eSeqFlags_LightAnimationSet;
 
     const unsigned int numNodes = animNodesRoot->getChildCount();
-    for (int i = 0; i < numNodes; ++i)
+    for (unsigned int i = 0; i < numNodes; ++i)
```

Comment:
I just checked this change in https://github.com/o3de/o3de/commit/ecd3b015eea1eb49bab8d0eb02f43b16054f82c4